src/components/10-atoms/button/index.js

```diff
@@ -145,7 +145,17 @@ class AXAButton extends InlineStyles {
     // form behaviour works (submit, reset, etc). The reason why it works with fake button is
     // that fake button is NOT inside a ShadowDOM. The event instead
     // bubbles out of ShadowDOM, hence the stop propagation trick
-    if (eventIsTrusted(e) && isNativeShadowDOM && this.isTypeSubmitOrReset) {
+
+    // IF statement that can be removed with IE11 support dropping
+    if (
+      document.documentMode &&
+      eventIsTrusted(e) &&
+      isNativeShadowDOM &&
+      this.isTypeSubmitOrReset
+    ) {
+      e.stopPropagation();
```

Comment:
DRY violation: the `if/else if` bodies have the same content, hence can be merged. Suggestion: make two named conditions `const isIESubmitResetEvent = ` and similar then do `if (isIESubmitResetEvent || isSubmitResetEvent) {...`